src/sync.h

```diff
@@ -71,6 +72,7 @@ template <typename MutexType>
 void static inline AssertLockHeldInternal(const char* pszName, const char* pszFile, int nLine, MutexType* cs) ASSERT_EXCLUSIVE_LOCK(cs) {}
 void static inline AssertLockNotHeldInternal(const char* pszName, const char* pszFile, int nLine, void* cs) {}
 void static inline DeleteLock(void* cs) {}
```

Comment:
```suggestion
void inline DeleteLock(void* cs) {}
```

Could remove the confusing static's here?

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/19340#issuecomment-667676282).